tests/phpstan/phpstan-latest.neon

```diff
@@ -2,8 +2,11 @@ includes:
 	- %currentWorkingDirectory%/tests/phpstan/phpstan.neon
 
 parameters:
+	excludes_analyse:
+		- ./../../classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php
 	ignoreErrors:
 		- '#Access to an undefined property Module::\$installed.#'
 		- '#Call to method fetchLocale\(\) on an unknown class PrestaShop\\PrestaShop\\Core\\Cldr\\Update.#'
 		- '#Function deactivate_custom_modules not found.#'
+		- '#Function deactivate_custom_modules80 not found.#'
```

Comment:
Instead of ignoring this one on each version of PHP wouldn't it be possible to update https://github.com/PrestaShop/autoupgrade/blob/1850ae27e0258850d95d7fc9b8c94da32295be73/tests/phpstan/phpstan.neon and add `deactivate_custom_modules.php` into its paths instead

I would even suggest adding all the `updagrade/php/*` scripts actually but it might have a lot of errors to handle which would be too much for this PR (it might be worth trying it and do it in another PR if it's too much)